tests/tests/main.fmf

```diff
@@ -6,3 +6,8 @@ test: |
     --log-cli-level=${LOG_LEVEL} \
     --log-cli-date-format="%Y-%m-%d %H:%M:%S%z" \
     --log-cli-format="%(asctime)s,%(msecs)03d %(levelname)-7s [%(name)s] %(message)s (%(module)s:%(lineno)d)"
+
```

Comment:
Could you please add following comment to describe why we have the `require` part?

```
# Require the whole tree (no pruning)
require:
   - type: file
     pattern:
       - /
```

Reply:
comment added